app/src/main/res/values/strings.xml

```diff
@@ -721,6 +721,36 @@
     <string name="label_reactions">Reactions</string>
     <string name="label_more_comming_soon">(more coming soon)</string>
     <string name="label_wait_until_synchronised">Please wait until the app is synchronised</string>
+    <string name="label_textfield_optional_password">PASSWORD (OPTIONAL)</string>
```

Comment:
Could we make it upper case on compose side ? we could miss that in other languages

Reply:
Done